java-sdk/src/main/java/org/sagebionetworks/bridge/sdk/BridgeAdminClient.java

```diff
@@ -35,9 +36,9 @@ public String createUser(StudyParticipant participant, boolean consentUser) {
 
         ObjectNode node = (ObjectNode)Utilities.getMapper().valueToTree(participant);
         node.put("consent", Boolean.toString(consentUser));
-        UserSession session = post(config.getUsersApi(), node, UserSession.class);
+        JsonNode sessionNode = postForJSON(config.getUsersApi(), node);
         
-        return session.getId();
+        return sessionNode.get("id").asText();
```

Comment:
See above, it's extracted to the StudyParticipant object which contains all the information about the user.